cli/cmd/compliance_aws.go

```diff
@@ -182,6 +186,30 @@ To run an ad-hoc compliance assessment of an AWS account:
 				)
 			}
 
+			// if a second arg if provided, we assume it's a recommendation id
+			if len(args) > 1 {
+				validRecommendation, _ := validateRecommendationID(args[1])
+				if validRecommendation {
+					recommendationID := args[1]
+					resources := filterResourcesByRecommendationID(report, recommendationID)
+
+					if len(resources) > 0 {
+						cli.OutputHuman("\nShowing '%d' resources affected by `%s`\n", len(resources), recommendationID)
+						cli.OutputHuman(
+							renderSimpleTable(
+								[]string{"Resource", "Region", "Reasons"},
+								resources,
+							),
+						)
+					} else {
+						cli.OutputHuman("\nNo resources found affected by `%s`\n", recommendationID)
```

Comment:
we use single quotes:
```suggestion
						cli.OutputHuman("\nNo resources found affected by '%s'\n", recommendationID)
```